nsxt/resource_nsxt_policy_service.go

```diff
@@ -184,6 +184,23 @@ func resourceNsxtPolicyService() *schema.Resource {
 					},
 				},
 			},
+
+			"nested_service_entry": {
+				Type:        schema.TypeSet,
+				Description: "Nested service service entry",
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"display_name": getOptionalDisplayNameSchema(false),
+						"description":  getDescriptionSchema(),
+						"nested_service_path": {
```

Comment:
I would suggest to call this `service_path` as `nested` is already part of the clause name `nested_service_entry`.
Also please use `getPolicyPathSchema` as schema definition since it also includes validation